docs/specs/legacy.yml

```diff
@@ -227,8 +227,8 @@ inputs:
 outputs:
   docs/a.json:
```

Comment:
would you please add one test case like below:
```
inputs:
  docfx.yml: |
    fileMetadata:
      author:
        '**/*': XXX
  docs/a.md:
outputs:

Reply:
sure